### PR TITLE
Add support for `-arch arch_type`

### PR DESCRIPTION
**Pull request by CRKatri**
This is identical to -A except that it uses the name of the arch not the
values of the CPU_TYPE/CPU_SUBTYPE.

This makes it more compatible with CC tools where this flag is very common.

### CODE REVIEW

File: docs/ldid.1 [hunk 1]
```diff
@@ -46,6 +45,16 @@ and
 and
 .Ar subtype
 should both be integers.
+.It Fl a
+Print the CPU types and subtypes in hexadecimal.
+.It Fl arch Ar arch_type
+The same as
+.Fl A ,
+except the name of the architecture is used.
+The list of currently know
```
Comment (asdfugil) [inline docs/ldid.1:54]:
```suggestion
The list of currently known
```

**Review by asdfugil — CHANGES_REQUESTED** [review]

**Review by asdfugil — CHANGES_REQUESTED** [review]
Please also update the help message to include this argument.

**Review by asdfugil — APPROVED** [review]

**Comment by CRKatri** [comment]
@asdfugil can you translate these manpage changes?

**Comment by asdfugil** [comment]
sure

**Comment by CRKatri** [comment]
You should have permission to push directly to this branch now, check your email.

**Comment by CRKatri** [comment]
Thanks

**Comment by asdfugil** [comment]
The readme is outdated and does not mention the change.

**Pull request merged**